backend/server.js

```diff
@@ -13,15 +16,178 @@ const port = process.env.PORT || 8080;
 const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors());
+app.use(
+  cors({
+    origin: "https://emilia-michelle-project-auth.netlify.app", 
+    methods: ["GET", "POST"], // Specify the allowed methods
+    allowedHeaders: ["Content-Type", "Authorization"], // Specify the allowed headers
+  })
+); 
 app.use(express.json());
 
+// Defines endpoint paths as constants to be able to only update the paths in one place if needed
+const PATHS = {
```

Comment:
Very smart solution! :) We will try to implement something similar in future projects. 
Nice way of working :D